site/readme.md

```diff
@@ -36,5 +35,49 @@ The web site is available at https://www.eclipse.org/hono.
 
 Publishing changes to the web site is done by means of pushing HTML files and other content to Hono's
 web site Git repository at the Eclipse Foundation.
-This is performed every night by the Eclipse build server using the Jenkins pipeline 
-script `~/hono/jenkins/Hono-Website-Pipeline.groovy`.
+This is performed every night by the Eclipse build server.
+
+## Release and Versioning
+
+For each supported version of Hono a separate version of the documentation is build. This is mostly automated by 
+Jenkins pipelines. 
+
+The build process of the documentation makes assumptions on the release of Hono. First of all the 
+[Semantic Versioning](https://semver.org/) schema is used for version numbers. For the purpose of the documentation 
+a version only includes MAJOR and MINOR level, meaning that the documentation should always reflect the latest 
+patch level of a MINOR (+ MAJOR) version of Hono.
+
+Every *newly* released MINOR (or MAJOR) version is considered to be the stable version.
+
+The files `site/homepage/versions_supported.csv` and `site/homepage/tag_stable.txt` determine which versions of the
+documentation are published. They are _written_ by the release pipeline (`jenkins/Hono-Release-Pipeline.groovy`) and 
+_read_ by the website pipeline (`jenkins/Hono-Website-Pipeline.groovy`).
+
+The website pipeline build the following versions:
+
+* `latest`: always build from the current `HEAD` of the master branch
+* each version in `site/homepage/versions_supported.csv`
+* `stable`: the version from the Git tag in the file `site/homepage/tag_stable.txt` (if the file exists and contains a Git tag)
+
+### What are the checkboxes in the release pipeline doing?
+
+If the checkbox "DEPLOY_DOCUMENTATION" is checked, the version that is being released is added (appended) to the file
```

Comment:
please do not use quotes to emphasize, use italic formatting instead, i.e. use `*DEPLOY_DOCUMENTATION*` instead